accessibility-checker/src/lib/ACConfigLoader.js

```diff
@@ -106,6 +106,9 @@ async function processACConfig(ACConfig) {
         try {
             ruleArchiveParse = await new Promise((resolve, reject) => {
                 request.get(ruleArchiveFile, function (error, response, body) {
+                    if (error) {
+                        reject(error);
+                    }
```

Comment:
We need an else here:
```
if (error) {
  reject(error);
} else {
  resolve(JSON.parse(body));
}
```

As-is, if there's an error, it will both reject and resolve.

Reply:
Thanks! This has been updated 👍